src/gar.lua

```diff
@@ -1206,18 +1206,22 @@ function gar.instantGatewayWithdrawal(from, gatewayAddress, vaultId, currentTime
 	assert(gateway, "Gateway not found")
 
 	local isGatewayWithdrawal = from == gatewayAddress
-	assert(gateway.status ~= "leaving", "This gateway is leaving and this vault cannot be instantly withdrawn.")
+
+	-- the protected operator vault is the vault that represents the gateway operators minimum stake and cannot be instantly withdrawn
+	local isGatewayProtectedVault = vaultId == gatewayAddress
 
 	local vault
 	local delegate
 	if isGatewayWithdrawal then
+		assert(gateway.vaults[vaultId], "Vault not found")
+		assert(not isGatewayProtectedVault, "Gateway operator vault cannot be instantly withdrawn.")
```

Comment:
Nit: error might be more precise with "Gateway operator minimum stake vault..."

Reply:
[3e3691c](https://github.com/ar-io/ar-io-network-process/pull/422/commits/3e3691ce53e6c0a818c49b732bcd8a12a9315e36)